packages/jaeger-ui/src/components/SearchTracePage/SearchForm.jsx

```diff
@@ -342,6 +342,17 @@ export class SearchFormImpl extends React.PureComponent {
                     <li>
                       Values containing whitespace or equal-sign &apos=&apos should be enclosed in quotes
                     </li>
+                    <li>
+                      Elasticsearch/OpenSearch storage requires escaping{' '}
```

Comment:
This doesn't seem to belong in the tool tip. The only escaping needed in the ui input is so that the ui parser can pick apart the right strings. If those strings still need additional escaping for storage, it should be handled by the storage implementation, transparently to the user.

Reply:
The ES storage uses on purpose regex terms (it regex support was added to ES storage some time ago). 

The goal here is to document this to let people know that escaping is needed for exact match queries.